packages/rest/test/helpers.ts

```diff
@@ -25,3 +30,16 @@ export function createUnexpectedHttpErrorLogger(
     );
   };
 }
+
+export function aBodySpec(
```

Comment:
👍 I love the idea to create a function generating a body spec, and we can extend the function to allow multiple contents in story https://github.com/strongloop/loopback-next/issues/1494